File: rating_problems/ratingproblems.cpp
#include <iostream>

int main() {
    int n, k, r, total=0;
    std::cin >> n >> k;
    for (int i=0; i<k; i++) {
        std::cin >> r;
        total += r;
    }
    float max_total = (float)(total + 3*(n-k))/n;
    float min_total = (float)(total - 3*(n-k))/n;

    std::cout << min_total << " " << max_total << std::endl;
}

File: rating_problems/ratingproblems.py
def main():
    n, k = map(int, input().split())
    total = 0
    for i in range(k):
        total += int(input())
    max_total = (total + (n-k)*3)/n
    min_total = (total - (n-k)*3)/n
    print(f"{min_total} {max_total}")


if __name__ == "__main__":
    main()